app/uk/gov/hmrc/soletraderidentificationfrontend/views/govuk_template.scala.html

```diff
@@ -17,43 +17,49 @@
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 @import uk.gov.hmrc.soletraderidentificationfrontend.views.helpers.FooterLinks
 @import uk.gov.hmrc.soletraderidentificationfrontend.config.AppConfig
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcHeader
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.Header
 
 @this(govukLayout: GovukLayout,
-      govUkHeader: GovukHeader,
-      appConfig: AppConfig,
-      footerLinks: FooterLinks,
-      footer: govukFooter)
+        hmrcHeader: HmrcHeader,
+        govUkHeader: GovukHeader,
```

Comment:
The GovukHeader isn't used anymore so we probably don't need it in this. You can then get rid of this import I think  as well @import uk.gov.hmrc.govukfrontend.views.html.components._